resinos-in-container.sh

```diff
@@ -98,6 +107,13 @@ if ! docker info &> /dev/null; then
     exit 1
 fi
 
+# Get absolute path of the script location
+# In this way we can reference any file relative to the script path easily
+# Get the absolute script location
+pushd "$(dirname "$0")" > /dev/null 2>&1
```

Comment:
You can also just do
```
SCRIPTPATH=$(cd $(dirname $0) ; pwd)
```

Reply:
Yeah. I just broke that for clarity. But I guess it wasn't a good idea.